### PR TITLE
Upgrade to React 16/Enzyme 3/Jest 22

### LINKED ISSUE #4: Warning about incorrect peer dependencies
Hi, when I install `enzyme-redux` with React 16.2 and Enzyme 3.2 I get this message:

```
warning " > enzyme-redux@0.1.7" has incorrect peer dependency "enzyme@^2.0.0".
warning " > enzyme-redux@0.1.7" has incorrect peer dependency "react@^15.0.0".
```

I don't see what makes this library incompatible with Enzyme 3 or React 16. Do yu think there is any downside in updating the constraints on these two dependencies?

### PR DESCRIPTION
**Pull request by schneidmaster**
Just bumped package versions and added Enzyme 3 setup configuration, no source changes needed

Fixes #4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,25 +24,25 @@
...
+    "react-dom": "^16.2.0",
+    "react-redux": "^5.0.7",
     "redux": "^3.5.2",
     "redux-test-utils": "^0.1.0",
     "rimraf": "^2.5.3"
   },
   "peerDependencies": {
-    "enzyme": "^2.0.0",
-    "react": "^15.0.0"
+    "enzyme": "^2 || ^3",
```
Comment (knegusen) [inline package.json]: Just say `^3` and `^16` instead.

**Review by knegusen — CHANGES_REQUESTED** [review]
Remove package-lock.json.

**Comment by schneidmaster** [comment]
@knegusen Updated, thanks

**Review by knegusen — APPROVED** [review]

**Comment by knegusen** [comment]
Thanks!

**Pull request merged**